app/views/chefs/index.html.erb

```diff
@@ -1,12 +1,9 @@
-<h1>Chefs#index</h1>
-<p>Find me in app/views/chefs/index.html.erb</p>
+<div class="container">
+<h1>Chefs</h1>
+</div>
+<div class="container">
```

Comment:
The current code lists the cuisine last name and price. Presume you will add this?